index.bs

```diff
@@ -5220,6 +5248,14 @@ In order to perform a [=registration ceremony=], the [=[RP]=] MUST proceed as fo
 
 1. Verify that the value of <code>|C|.{{CollectedClientData/origin}}</code> matches the [=[RP]=]'s [=origin=].
 
+1. If <code>|C|.{{CollectedClientData/topOrigin}}</code> is present:
```

Comment:
Multiple references to step 19 and 20 below in paragraph 7.1 should be (manually?) updated.

Reply:
Thanks! Moved to new issue: #1861